app/qml/components/MMMorePhoto.qml

```diff
@@ -0,0 +1,105 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import Qt5Compat.GraphicalEffects
+import "."
```

Comment:
Given that style is now read from`__style` we do not need to `import "."`, right?